backend/src/apiserver/resource/resource_manager.go

```diff
@@ -97,6 +100,13 @@ func NewResourceManager(clientManager ClientManagerInterface) *ResourceManager {
 	}
 }
 
+func (r *ResourceManager) GetWorkflowClient(namespace string) workflowclient.WorkflowInterface {
+	if len(namespace) == 0 {
+		return r.argoClient.Workflow(PodNamespace)
```

Comment:
how does this work since PodNamespace= "POD_NAMESPACE"?

Reply:
This was an outdated commit. Could you check the new commit? It should've been fixed.